lib/swaggerUtil.js

```diff
@@ -103,16 +97,16 @@ function validateObject(specs, obj, disc) {
     var validationErrors = [];
     if (!obj[disc]) {//discriminator not defined for this object
         validationErrors.push(
-                {
-                    message: 'discriminator ' + disc + ' not defined for inheriting model ' + JSON.stringify(obj)
-                }
+            {
+                message: 'discriminator ' + disc + ' not defined for inheriting model ' + JSON.stringify(obj)
+            }
             );
     }
     else if (!specs.definitions[obj[disc]]) {
         validationErrors.push(
-                {
-                    message: 'model definition not found for ' + disc + ". ALL model definitions must be defined in root swagger document"
-                }
+            {
+                message: 'model definition not found for ' + disc + '. ALL model definitions must be defined in root swagger document'
```

Comment:
it's strange that there's a period in this message ... suggest that it change to a colon (`:`)